### PR TITLE
qbo-components testing commit

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/app/core/interceptors/jwt.interceptor.spec.ts [hunk 1]
```diff
@@ -1,22 +1,173 @@
...
+  let authService: AuthService;
+  const API_BASE_URL = environment.api_url;
+  const workspace_id = environment.tests.workspaceId;
+  const response = {
+    access_token: "fylee",
+    expires_in: 3600,
+    refresh_token: "ffff",
+    token_type: "Bearer"
+  };
+  const response1: Token = {
```
Comment (ashwin1111) [inline src/app/core/interceptors/jwt.interceptor.spec.ts:28]: add these stuffs in fixture, makes the spec file much cleaner

Comment (ashwin1111) [reply src/app/core/interceptors/jwt.interceptor.spec.ts:28]: consider this change across all the files in the PR

**Review by ashwin1111 — APPROVED** [review]

**Comment by ashwin1111** [comment]
Please fix the 1 failed test as well
```
TypeError: Cannot read properties of null (reading 'id')
    at MatSelect._compareWith (http://localhost:9876/_karma_webpack_/webpack:/src/app/core/services/core/helper.service.ts:24:136)
    at http://localhost:9876/_karma_webpack_/webpack:/node_modules/@angular/material/fesm2015/select.mjs:678:53
    at Array.find (<anonymous>)
    at QueryList.find (http://localhost:9876/_karma_webpack_/webpack:/node_modules/@angular/core/fesm2015/core.mjs:22613:1)
    at MatSelect._selectOptionByValue (http://localhost:9876/_karma_webpack_/webpack:/node_modules/@angular/material/fesm2015/select.mjs:670:50)
    at MatSelect._setSelectionByValue (http://localhost:9876/_karma_webpack_/webpack:/node_modules/@angular/material/fesm2015/select.mjs:651:46)
    at http://localhost:9876/_karma_webpack_/webpack:/node_modules/@angular/material/fesm2015/select.mjs:632:18
    at ZoneDelegate.invoke (http://localhost:9876/_karma_webpack_/webpack:/node_modules/zone.js/fesm2015/zone.js:372:1)
    at ProxyZoneSpec.onInvoke (http://localhost:9876/_karma_webpack_/webpack:/node_modules/zone.js/fesm2015/zone-testing.js:287:1)
    at ZoneDelegate.invoke (http://localhost:9876/_karma_webpack_/webpack:/node_modules/zone.js/fesm2015/zone.js:371:1)

File: src/app/core/interceptors/jwt.interceptor.spec.ts (hunk 1 above)
Comment (DhaaraniCIT) [reply src/app/core/interceptors/jwt.interceptor.spec.ts:28]: these are same across the project

**Pull request merged**